android/java/res/layout/brave_wallet_add_network.xml

```diff
@@ -219,20 +219,21 @@
             android:id="@+id/add"
             android:layout_width="wrap_content"
             android:layout_height="0dp"
-            android:layout_gravity="start"
+            android:layout_gravity="center_horizontal"
             android:gravity="center"
             android:layout_weight = "1"
             android:background="@drawable/crypto_wallet_blue_button"
             android:text="@string/brave_wallet_add_network_add"
             android:textAllCaps="false"
             android:textSize="16sp"
             android:layout_marginTop="10dp"
-            android:layout_marginStart="24dp"
+            android:layout_marginStart="12dp"
             android:layout_marginEnd="12dp"
+            android:layout_marginBottom="12dp"
             android:paddingTop="8dp"
             android:paddingBottom="8dp"
-            android:paddingStart="16dp"
-            android:paddingEnd="16dp"
+            android:paddingStart="8dp"
+            android:paddingEnd="8dp"
```

Comment:
nit: can be replaced with [android:paddingHorizontal](https://developer.android.com/reference/android/view/View#attr_android:paddingHorizontal)